lib/taskana-simplehistory-rest-spring/src/main/java/pro/taskana/rest/simplehistory/TaskHistoryEventController.java

```diff
@@ -409,4 +409,33 @@ private HistoryQuery applyFilterParams(HistoryQuery query,
 
         return query;
     }
+
+    private TimeInterval getTimeIntervalFor(String created) {
+        LOGGER.debug("Date-String is: {}", created);
```

Comment:
I would recommend you to define a default date type like, dd-MM-yyyy or MM-dd-yyyy  and if the parameter doesn't match just throw a descriptive erro message